horace_core/algorithms/change_crystal_horace.m

```diff
@@ -1,49 +1,46 @@
 function varargout=change_crystal_horace(varargin)
-% Change the crystal lattice and orientation in a file or set of files containing a Horace data object
-% 
-% Most commonly:
-%   >> change_crystal (file, rlu_corr)              % change lattice parameters and orientation
+% Change the crystal lattice and orientation of an sqw/dnd objects or
+% sqw/dnd object  stored in a file or celarray of files
```

Comment:
there is no indication here or elsewhere why we should have this function as well as the others present in this PR. We already have change crystal_sqw, change_crystal_dnd, and change_crystal. It would be good to have lots of documentation saying when each of these should be used for preference. If one of them (guessing this one) is for back compatibility, this should be made clear.

Reply:
well, this is PR about dealing with `change_crystal` family. And this one is part of the family.